boards/teensy31/include/board.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_teensy31 Teensy3.1 & 3.2
```

Comment:
This line should be removed and the line below updated: `@ingroup boards_teensy31`